src/ai-bundle/src/AiBundle.php

```diff
@@ -750,8 +750,8 @@ private function processStoreConfig(string $type, array $stores, ContainerBuilde
 
         if ('clickhouse' === $type) {
             foreach ($stores as $name => $store) {
-                if (isset($store['http_client'])) {
-                    $httpClient = new Reference($store['http_client']);
+                if (isset($store['client'])) {
+                    $httpClient = new Reference($store['client']);
```

Comment:
needed?

Reply:
rah, that's a typo. Sorry. (I understand now why I had to change twice a piece of code 🤦🏼, haha)